www/caddy/src/opnsense/scripts/OPNsense/Caddy/caddy_certs.php

```diff
@@ -29,6 +29,8 @@
  */
 
 require_once("config.inc");
+require_once("certs.inc");
```

Comment:
at first glance I don't see anything from certs.inc... should work without it? I was only worried about legacy_bindings sine you are using `OPNsense\Core\Config`

Reply:
Yeah I have tested the script without those, and it worked. I thought you knew something more than me and thus readded it again. Better safe than sorry.

Reply:
Hmm we need https://github.com/opnsense/core/blob/f7ac887d905529b88475c6e41aebfbfaa0cb68b3/src/etc/inc/legacy_bindings.inc#L29 but apparently config.inc already includes legacy_bindings.inc so you can remove again... sorry for the noise

Reply:
Hey it's okay, thanks for looking over it. ^^